src/main/webapp/app/detail-overview-list/detail-overview-list.component.html

```diff
@@ -4,7 +4,7 @@
 @for (section of sections; track section) {
     <h3 class="section-headline" [id]="headlinesRecord[section.headline]">{{ section.headline | artemisTranslate }}</h3>
     <dl class="section-detail-list">
-        @for (detail of section.details; track detail) {
+        @for (detail of section.details; track $index) {
```

Comment:
Was this change intentional for this PR?

Reply:
Yes, this is actually a bugfix. Interestingly, you only encounter the issue locally though. When you go to the execise-details page, this warning is constantly triggered:

![image](https://github.com/user-attachments/assets/ffcad1b1-26c6-4cc6-8742-f12d5b6a67ef)